docker-compose/install.sh

```diff
@@ -26,28 +47,28 @@ progress() {
     printf "\r\e[K|%-*s| %3d%% %s\n" "50" "$char" "$1" "${@:2}";
 }
 
-write_compose_file() {
+write_compose_file() {  # TODO: update this with latest image tags + correct environment variables
```

Comment:
please pull the compose file instead of hardcoding it here. The other developers will not be maintaining this file, so it will desynchronize very quickly

Reply:
This function is only called when the script is directly downloaded and the git clone fails. The thing is I just keep the code in case the original uploader of the script was thinking on other uses, but right now the script always gets a fresh pulled docker-compose file. Anyways, I'm removing it to avoid any confusion.